resources/assets/js/app/collections/userVotesCollection.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Created by lyudmila on 26.08.16.
+ */
+var baseCollection = require('../instances/Collection');
+var baseModel = require('../instances/Model');
+
+module.exports = baseCollection.extend({
+    url: '/votes',
+    model: baseModel,
+    initialize: function (models, options) {
+        if (options.parentUrl) {
+            this.parentUrl = options.parentUrl;
+        }
+    }
+});
```

Comment:
you no need new VoteCollection^ you can use already created VoteCollection class